diagnostic.md

```diff
@@ -35,7 +36,12 @@ Record your responses inside the fenced code blocks below each question.
     ```
 
     ```md
-    <!-- your response here -->
+      1. top one
+        1. url is `asdf.com/products/:product_id`
+        2. it will return the data of one spefic product with X id
+      2. Bottom
+        1. url is `asdf.com/products/:product_id`
+        2. it will return the same data
```

Comment:
You are correct that these two route definitions are identical  URL paths.  What differences do you see between these route definitions?  One of the biggest distinctions is where Ember will render the content.  
The nested `'products'/'product'` instructs Ember to render
`app/products/product/template.hbs` in the `outlet` of the parent `app/products/template.hbs`.  
The not-nested `'product'` instructs Ember to render `app/product/template.hbs` in the context of the application template.